docusaurus/docs/Android/04-compose/04-message-components/03-message-list.mdx

```diff
@@ -66,35 +81,44 @@ fun MessageList(
 	..., // State & UI
     onThreadClick: (Message) -> Unit = { viewModel.openMessageThread(it) },
     onLongItemClick: (Message) -> Unit = { viewModel.selectMessage(it) },
+    onReactionsClick: (Message) -> Unit = { viewModel.selectReactions(it) },
     onMessagesStartReached: () -> Unit = { viewModel.loadMore() },
     onLastVisibleMessageChanged: (Message) -> Unit = { viewModel.updateLastSeenMessage(it) },
-    onScrolledToBottom: () -> Unit = { viewModel.clearNewMessageState() },
+    onScrollToBottom: () -> Unit = { viewModel.clearNewMessageState() },
+    onGiphyActionClick: (GiphyAction) -> Unit = { viewModel.performGiphyAction(it) },
     onImagePreviewResult: (ImagePreviewResult?) -> Unit = {
         if (it?.resultType == ImagePreviewResultType.SHOW_IN_CHAT) {
             viewModel.focusMessage(it.messageId)
         }
     },
+    ... // Content
 )
 ```
 
 * `onThreadClick`: Handler for the user tapping on a message with a thread.
```

Comment:
Gerund and simple present is mixed here. Maybe one approach across all params will be better?

Reply:
Handler when the user taps should be good

Reply:
I added used before when because I feel without it it doesn't imply causality strongly enough.